sql/api/src/main/scala/org/apache/spark/sql/catalyst/util/RebaseDateTime.scala

```diff
@@ -22,7 +22,7 @@ import java.time.temporal.ChronoField
 import java.util.{Calendar, TimeZone}
 import java.util.Calendar.{DAY_OF_MONTH, DST_OFFSET, ERA, HOUR_OF_DAY, MINUTE, MONTH, SECOND, YEAR, ZONE_OFFSET}
 
-import scala.collection.mutable.AnyRefMap
+import scala.collection.mutable
```

Comment:
```suggestion
import scala.collection.mutable.HashMap
```

Reply:
Updated.

Reply:
Aside from the original goals of the PR itself, personally, I tend to write it as follows:

```
import scala.collection.mutable
val hashMap = new mutable.HashMap[String, RebaseInfo]
```

rather than

```
import scala.collection.mutable.HashMap
val hashMap = new HashMap[String, RebaseInfo]
```


This way, when we need to import `immutable.HashMap` in this class, it won't cause a naming conflict (generally, for immutable data structures, there's no need to add an extra prefix). Additionally, using `mutable.HashMap` makes the code clearer when reading it."